### ISSUE TITLE
Middlebury dataset

### ISSUE THREAD

**Issue by yyYcoco2** [gengshan-y/high-res-stereo#49]
after download gooledrive middlebury zip，it has md-ex and md -ex training，which one is training set，which one is validation set
